src/main/resources/assets/css/marathon.css

```diff
@@ -157,10 +157,6 @@ a,
   border-color: #3d4045;
 }
 
-.list-inline > li {
-  margin-right: 10px;
-}
-
 .list-inline-subtext {
```

Comment:
They remove the padding to the right of the list-inline last `li` child and and the bottom margin of the `ul`. The bottom margin becomes apparent when you have multiple deployments taking place at the same time.